integration-fe/src/sagas/getBaseWebHookURL.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-debugger */
+import { call, put } from 'redux-saga/effects';
+import { getBaseWebHookURL as getURL } from './apiCalls';
```

Comment:
The same as the above. Each "saga" file has its own role. This returns the base webhook url from the url params.